configs/sst_virtualization-virt.yaml

```diff
@@ -0,0 +1,97 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization on RHEL - KVM Userspace
+  description: Userspace agents and tools that enable virtualization capabilities on RHEL
+  maintainer: sst_virtualization
+
+  modules_enable:
+  - virt:rhel
+
+  packages:
+  - amtterm
+  - augeas
+  - dtc
+  - edk2
+  - fakechroot
+  - fakeroot
+  - fuse-sshfs
+  - gperftools
+  - gtk-vnc
+  - hexedit
+  - hivex
+  - icoutils
+  - ipxe
+  - kernel
+  - libdnet
+  - libguestfs
+  - libguestfs-winsupport
+  - libiscsi
+  - libmspack
+  - libnbd
+  - libosinfo
+  - libunwind
+  - libvirt
+  - libvirt-dbus
+  - libvirt-glib
+  - libvirt-python
+  - libvirt-snmp
+  - mdevctl
+  - mingw-binutils
+  - mingw-crt
+  - mingw-filesystem
+  - mingw-gcc
+  - mingw-headers
+  - mingw-pkg-config
+  - mingw-srvany
+  - mingw-w64-tools
```

Comment:
None of the mingw stuff should be listed under virt

Reply:
But should they be in RHEL-9 ?

If yes, we need to ask the sst_desktop (or the right SST) to take over ownership and mention these in their Content Set. 
If no, they will go in the  exclusion list.

Reply:
Note that `mingw-srvany` is not like the other packages.  It is a binary that is used by libguestfs and virt-v2v.

Reply:
So in addition to what I said above, please don't remove `mingw-srvany`.  You can assign it to me if you like.